### PR TITLE
chore: add `LICENSE`

### LINKED ISSUE #142: Missing License
### Have you experienced this bug with the latest version of the template?

yes

### Steps to Reproduce

Check the repo and all surrounding information. This code has no license.

### Expected Behavior

There should be a license, otherwise all of this code is automatically "all rights reserved" under US law, as far as I'm aware.

### Actual Behavior

This code is unlicensed and inappropriate for anyone to use.

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Closes #142

### CODE REVIEW

File: remix.init/index.js [hunk 1]
```diff
@@ -238,6 +238,7 @@ const main = async ({ isTypeScript, packageManager, rootDirectory }) => {
     fs.rm(path.join(rootDirectory, ".github", "workflows", "no-response.yml")),
     fs.rm(path.join(rootDirectory, ".github", "dependabot.yml")),
     fs.rm(path.join(rootDirectory, ".github", "PULL_REQUEST_TEMPLATE.md")),
+    fs.rm(path.join(rootDirectory, "LICENSE.md")),
```
Comment (brophdawg11) [inline remix.init/index.js:241]: Does this not go against this portion of the license?

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Comment (brophdawg11) [reply remix.init/index.js:241]: I guess that applies more to if you clone the blues stack, not install from it?

Comment (MichaelDeBoey) [reply remix.init/index.js:241]: > I guess that applies more to if you clone the blues stack, not install from it?

That's indeed my thought as well

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Remix Software 2022
```
Comment (MichaelDeBoey) [inline LICENSE.md:3]:
```suggestion
Copyright (c) Shopify Inc. 2022
```

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Shopify Inc. 2022
```
Comment (mjackson) [inline LICENSE.md:3]:
```suggestion
Copyright (c) Remix Software Inc. 2021
Copyright (c) Shopify Inc. 2022-2023
```

**Pull request merged**